### ISSUE TITLE
Remark problem

### ISSUE THREAD

**Issue by hsnkvk** [SafranCassiopee/csharp-metar-decoder#8]
Hi. I think this code is not supporting remarks at the end of Metar messages. Is there any task that you are working on it?

**Comment by laurentkoka** [comment]
Hi. Correct. 

The OACI specs for the RMK field do not say much:

https://library.wmo.int/opac/doc_num.php?explnum_id=4157
"15.15 Group (RMK . . . . . . . . . . )
The indicator RMK denotes the beginning of a section containing information included
by national decision which shall not be disseminated internationally."
 
The same discussion occurred on the page of our PHP METAR decoder : 
https://github.com/SafranCassiopee/php-metar-decoder/issues/4

We will consider it again and keep you updated.

**Comment by hsnkvk** [comment]
Thanks for the reply.
Actually I've wrote code to your library for my country standards while I was waiting for your reply. But as you said it is a local acceptance and I'm not sure it's fully covering all possible messages. It won't be useful for somebody even I contribute to your project. 
This is an example of a remark that I achieved to parse:
".....RMK RWY17L 00000KT RWY05 09005KT 040V170 RWY23 12006KT 080V160"
